fixtures/fileOperations.ts

```diff
@@ -62,17 +62,32 @@ export class FileOperations {
     }
   }
 
-  async copyFileToTestDestination(page: Page) {
+  async goToTestStartingFolder(page: Page, location: string, portal: string, projectid: string) {
```

Comment:
I took out `portal` here and it didn't affect the tests in any way I could see.
```suggestion
  async goToTestStartingFolder(page: Page, location: string, projectid: string) {
```